Bwt/BwtProj/Bwt.cs

```diff
@@ -0,0 +1,54 @@
+﻿namespace BWT;
+/// <summary>
```

Comment:
```suggestion
﻿namespace BWT;
﻿
/// <summary>
```
Крупные логические куски структуры файла разделяются пустой строкой